packages/react-reconciler/src/ReactFiberLane.js

```diff
@@ -513,7 +513,7 @@ export function includesExpiredLane(root: FiberRoot, lanes: Lanes): boolean {
   return (lanes & root.expiredLanes) !== NoLanes;
 }
 
-export function isTransitionLane(lane: Lane): boolean {
+export function includesTransitionLane(lane: Lane): boolean {
```

Comment:
Can you rename this back please? It's meant to only accept the Lane type, not Lanes. Flow doesn't error if you pass it a Lanes because they are both aliases for `number` but we should make them opaque types so that Flow can enforce it. (They were originally opaque types, but we had to remove the opaqueness when we forked the reconciler, but now that the reconciler fork no longer exists we should make them opaque again.)

Reply:
makes sense, I misread the flow type